doc/tutorial/using_multi_gpu.txt

```diff
@@ -0,0 +1,127 @@
+
+.. _using_multi_gpu:
```

Comment:
I would say that this is very inconsistent since more than half of the files in tutorial/ don't have any sort of prefix and those that do don't all have the same form (it can be one of 'tut', 'tut_', 'tutorial', 'tutorial_') or have it somewhere in the middle.

Is this something we want to address?